public/index.html

```diff
@@ -8,6 +8,7 @@
     <title>TODO LIST</title>
 </head>
 <body>
+    <noscript>You need to enable JavaScript to run this app.</noscript>
     <div id="root"></div>
 </body>
 </html>
```

Comment:
Add an empty line at the end of the file in your editor

Reply:
added